firmware/quadruna/FSM/src/tasks.c

```diff
@@ -240,7 +247,10 @@ _Noreturn void tasks_runCanTx(void)
     for (;;)
     {
         CanMsg tx_msg = io_canQueue_popTx();
-        io_can_transmit(&can, &tx_msg);
+        if (!hw_can_transmit(&can, &tx_msg))
+        {
+            LOG_ERROR("CAN TX failed!");
+        }
```

Comment:
Maybe do this for all the boards? seems kinda weird to only log the error on some

Reply:
oh yes i forgor